workflows/theiacov/wf_theiacov_ont.wdl

```diff
@@ -427,5 +427,7 @@ workflow theiacov_ont {
     # QC_Check Results
     String? qc_check = qc_check_task.qc_check
     File? qc_standard = qc_check_task.qc_standard
+    # Non-flu specific outputs
+    String percentage_mapped_reads = select_first([stats_n_coverage.percentage_mapped_reads, flu_track.percentage_mapped_reads, ""])
```

Comment:
could you also make this check the _primtrim one first, and then try the regular stats_n_coverage next?

Reply:
added primtrim version first